CHANGELOG.md

```diff
@@ -11,10 +11,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
-- [Introduce conditionals on `Element`](https://github.com/square/Blueprint/pull/198) to allow you to perform inline checks like `if`, `if let`, and `map` when building `Element` trees.
```

Comment:
I think I made the right call, as this also appears in Past Releases. Please let me know if I should revert.

Reply:
Ah yep, looks like a merge conflict. Thanks!